ui/components/app/transaction-decoding/transaction-decoding.component.js

```diff
@@ -124,9 +135,20 @@ export default function TransactionDecoding({ to = '', inputData: data = '' }) {
 
           case 'uint':
             return (
-              <span className="sol-item solidity-uint">
-                {[value.asBN || value.asString].toString()}
-              </span>
+              <Tooltip
+                title={[value.asBN || value.asString].toString()}
+                interactive="true"
```

Comment:
Can this be just the attribute name `interactive`?  I think with React you don't need to add the `"true"` in this case.